frontend/src/scenes/insights/EmptyStates/EmptyStates.tsx

```diff
@@ -199,79 +266,66 @@ export function StatelessInsightLoadingState({
         return () => clearInterval(interval)
     }, [])
 
-    const bytesPerSecond = (bytesRead / (secondsElapsed || 1)) * 1000
-    const estimatedRows = pollResponse?.status?.query_progress?.estimated_rows_total
-
-    const cpuUtilization =
-        (pollResponse?.status?.query_progress?.active_cpu_time || 0) /
-        (pollResponse?.status?.query_progress?.time_elapsed || 1) /
-        10000
-
     const suggestions = suggestion ? (
         suggestion
-    ) : (
+    ) : showLoadingDetails ? (
         <div className="flex gap-3">
             <p className="text-xs m-0">Need to speed things up? Try reducing the date range.</p>
         </div>
-    )
+    ) : null
 
     return (
         <div
             data-attr="insight-empty-state"
-            className={clsx('flex flex-col gap-1 rounded p-4 w-full h-full', {
+            className={clsx('flex flex-col rounded p-4 w-full h-full', {
                 'justify-center items-center': !renderEmptyStateAsSkeleton,
-                'insights-loading-state justify-start': renderEmptyStateAsSkeleton,
+                'insights-loading-state': renderEmptyStateAsSkeleton,
```

Comment:
For a skeleton it made more sense to render that left-aligned, like Cory designed. Any reason why you centered it now?

Reply:
I'm not sure where the skeleton loader shows up tbh, I don't see it on dashboards or regular insight loading. But I've moved it left again!

Reply:
Only on Web Vitals and the new SQL editor at the moment